src/index.html

```diff
@@ -1,17 +1,200 @@
+        <label class="form-field">
+          Full date of birth:
+          <input 
+          type="date" 
+          name="date"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input 
+          type="checkbox" 
+          name="accept"
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset 
+      class="group"
+      >
+        <legend>
+          Registration
+        </legend>
+        <label 
+        class="form-field"
+        >
+          E-mail:
+          <input 
+          type="email" 
+          name="E-mail" 
+          placeholder="email@example.com"
+          >
+        </label>
+
+        <label>
+          Password:
+          <input 
+          type="password" 
+          name="Password" 
+          required
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset class="group">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <label 
+        class="form-field"
+        >
+          Do you love cats?
+          <input 
+          type="radio" 
+          name="radio" 
+          id="radio01"
+          >
+          Yes
+          <input 
+          type="radio" 
+          name="radio" 
+          id="radio02"
+          >
+          No
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input 
+          type="color" 
+          name="color"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input 
+          type="time" 
+          name="time"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select 
+          name="cars" 
+          multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label>
+          How do you rate our work?
+          <input 
+          type="range" 
+          name="rate" 
+          max="100" 
+          min="0" 
+          value="0"
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset 
+      class="group">
+        <legend>
+          Additional info
+        </legend>
+        <label 
+        class="form-field"
+        >Comments:
+          <textarea 
+          name="textarea" 
+          cols="20" 
+          rows="2"
+          >
+          </textarea>
+        </label>
+
+        <label>
+          Would you recommend us?
+          <select name="recommend">
+            <option value="+">yes</option>
+            <option value="-">no</option>
+          </select>
+        </label>
+      </fieldset>
+
+
+      <button type="submit">
+        Submit
+      </button>
+    </form>
+    <script 
+    type="text/javascript" 
+    src="./main.js"
+    >
+    </script>
   </body>
-</html>
+
+</html>
```

Comment:
Don't add empty lines between parent and children
```suggestion
</html>

```